Core/Core/API/APIModule.swift

```diff
@@ -22,4 +22,74 @@ public struct APIModule: Codable, Equatable {
     let name: String
     let position: Int
     let published: Bool
+    let items: [APIModuleItem]?
+}
+
+// https://canvas.instructure.com/doc/api/modules.html#ModuleItem
+public struct APIModuleItem: Codable, Equatable {
+    let id: ID
+    let moduleID: ID
+    /// The position of this item in the module (1-based)
+    let position: Int
+    let title: String
+    /// 0-based indent level; module items may be indented to show a hierarchy
+    let indent: Int
+    let content: ModuleItemType
+    let htmlURL: URL
+    /// (Optional) link to the Canvas API object, if applicable
+    let url: URL?
+    /// Only present if the caller has permission to view unpublished items
+    let published: Bool?
+
+    public enum CodingKeys: String, CodingKey {
```

Comment:
It seems common for our API models to be snake case. Is there a reason we don't want `moduleID` to be `module_id` and `htmlURL` to be `html_url` and remove the need for the `CodingKeys`?

Reply:
Ah, I forgot about that. Good call.

I still need `CodingKeys` because I'm doing custom decoding and encoding. But I do want casing to be consistent.